client/src/components/navbar/navbarForm.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Field, reduxForm } from 'redux-form'
+import { RaisedButton, TextField } from 'material-ui'
+import './style.css'
+
+const renderTextField = ({ input }) => (
+  <div className="navbarSearch">
+    <TextField hintText="Search" {...input} />
+  </div>
+)
+
+const SearchBar = props => {
+  const { handleSubmit } = props
```

Comment:
We should create a container for the form instead of passing fetchResults from the navbar